custom/auth/ante/min_initial_deposit.go

```diff
@@ -0,0 +1,75 @@
+package ante
+
+import (
+	"fmt"
+	//"strconv"
```

Comment:
Might as well just remove this import if its not being used :)

Reply:
I actually need that import. So can we keep it?

Reply:
![image](https://user-images.githubusercontent.com/29800180/221577333-758bcaf8-04bc-4d72-8de5-c6209849ba68.png)

Reply:
I was thinking about "strconv" which is commented out at the moment :)

Reply:
ok - removed it